<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3000</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="">
        <property name="title">
         <string>Codec</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="cp1251radioButton">
           <property name="text">
            <string>cp1251</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">codecButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="utf8radioButton">
           <property name="text">
            <string>UTF-8</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">codecButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="">
        <property name="title">
         <string>Font Style</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QCheckBox" name="boldCheckBox">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Bold</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">fontStyleButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="italicCheckBox">
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Italic</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">fontStyleButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="underlineCheckBox">
           <property name="font">
            <font>
             <underline>true</underline>
            </font>
           </property>
           <property name="text">
            <string>Underline</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">fontStyleButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="strikeoutCheckBox">
           <property name="font">
            <font>
             <strikeout>true</strikeout>
            </font>
           </property>
           <property name="text">
            <string>Strikeout</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">fontStyleButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="">
        <property name="title">
         <string>Text Color</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QComboBox" name="colorComboBox">
           <item>
            <property name="text">
             <string>Black</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Green</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Blue</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Red</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Violet</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTreeView" name="dirTreeView"/>
      </item>
      <item>
       <widget class="QListView" name="fileListView"/>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="font">
         <font>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="codecButtonGroup"/>
  <buttongroup name="fontStyleButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
